src/gui/guiVolumeChange.cpp

```diff
@@ -97,24 +97,24 @@ void GUIVolumeChange::regenerateGui(v2u32 screensize)
 				true, this, ID_soundText);
 	}
 	{
-		core::rect<s32> rect(0, 0, 80, 30);
-		rect = rect + v2s32(size.X/2-80/2, size.Y/2+55);
+		core::rect<s32> rect(0, 0, 80 * s, 30 * s);
+		rect = rect + v2s32(size.X / 2 - (80 * s) / 2, size.Y / 2 + 55 * s);
```

Comment:
Inconsistent and unnecessary brackets around `80 * s`

Reply:
Indeed, I originally had it all that way until I properly remembered the 'bidmas' rule, well spotted!